packages/core/src/Stores/traders-hub-store.js

```diff
@@ -659,6 +706,85 @@ export default class TradersHubStore extends BaseStore {
         this.is_account_transfer_modal_open = !this.is_account_transfer_modal_open;
     }
 
+    async updatePlatformBalance() {
+        runInAction(() => {
+            this.is_balance_calculating = true;
+        });
+        const { accounts, dxtrade_accounts_list, mt5_login_list, derivez_accounts_list } = this.root_store.client;
+
+        const account_list = Object.keys(accounts).map(loginid => accounts[loginid]);
+        const platform_demo_account = account_list.find(account => account.is_virtual);
+        if (platform_demo_account) {
+            const { balance, currency } = platform_demo_account;
+            this.platform_demo_balance = { balance, currency };
+        }
+
+        const platform_real_accounts = account_list.filter(
+            account =>
+                !account.is_virtual &&
+                (this.is_eu_user
+                    ? account.landing_company_shortcode === 'maltainvest'
+                    : account.landing_company_shortcode !== 'maltainvest')
+        );
+        if (platform_real_accounts?.length) {
+            this.platform_real_balance = await this.getTotalBalance(
+                platform_real_accounts,
+                platform_real_accounts[0].currency
+            );
+        }
+
+        let cfd_accounts = [];
+        if (Array.isArray(mt5_login_list)) {
+            cfd_accounts = [...cfd_accounts, ...mt5_login_list];
+        }
+        if (Array.isArray(dxtrade_accounts_list)) {
+            cfd_accounts = [...cfd_accounts, ...dxtrade_accounts_list];
+        }
+        if (Array.isArray(derivez_accounts_list)) {
+            cfd_accounts = [...cfd_accounts, ...derivez_accounts_list];
+        }
+
+        const cfd_real_accounts = cfd_accounts.filter(
+            account =>
+                account.account_type === 'real' &&
+                (this.is_eu_user
+                    ? account.landing_company_short === 'maltainvest'
+                    : account.landing_company_short !== 'maltainvest')
+        );
+        if (cfd_real_accounts?.length) {
+            this.cfd_real_balance = await this.getTotalBalance(cfd_real_accounts, cfd_real_accounts[0]?.currency);
+        } else {
+            this.cfd_real_balance = { balance: 0, currency: 'USD' };
+        }
+
+        const cfd_demo_accounts = cfd_accounts.filter(account => account.account_type === 'demo');
+        if (cfd_demo_accounts?.length) {
+            this.cfd_demo_balance = await this.getTotalBalance(cfd_demo_accounts, cfd_demo_accounts[0]?.currency);
+        }
+
+        runInAction(() => {
+            this.is_balance_calculating = false;
+        });
+    }
+
+    async getTotalBalance(accounts, base_currency) {
```

Comment:
@shontzu-deriv Are these changes intentional? 🤔 It's all been removed in #8147 👀 

@shayan-deriv Would you please help to check this 🙏🏻